packages/protocol/contracts/layer1/preconf/iface/IPreconfRouter2.sol

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import "src/layer1/based/IProposeBatch.sol";
+import "src/layer1/based/ITaikoInbox.sol";
+
+/// @title IPreconfRouter2
+/// @custom:security-contact security@taiko.xyz
+interface IPreconfRouter2 is IProposeBatch {
+    error ForcedInclusionNotSupported();
+    error InvalidCurrentLookahead();
```

Comment:
what if we move all errors to the implementation to avoid defining IPreconfRouter2?

Actually I feel like most errors are very implementation specific, maybe we should move all of them from interfaces to implementations.